{{cookiecutter.project_slug}}/Makefile

```diff
@@ -56,21 +57,21 @@ help:
 
 .PHONY: setup
 setup: ### Setup remote environment
-	neuro kill $(SETUP_NAME) >/dev/null 2>&1
-	neuro run \
+	$(NEURO) kill $(SETUP_NAME) >/dev/null 2>&1
+	$(NEURO) run \
 		--name $(SETUP_NAME) \
 		--preset cpu-small \
 		--detach \
 		--volume $(PROJECT_PATH_STORAGE):$(PROJECT_PATH_ENV):ro \
 		--env PLATFORMAPI_SERVICE_HOST="." \
 		$(BASE_ENV_NAME) \
 		'sleep 1h'
-	neuro cp $(REQUIREMENTS_APT) $(REQUIREMENTS_APT_STORAGE)
-	neuro cp $(REQUIREMENTS_PIP) $(REQUIREMENTS_PIP_STORAGE)
-	neuro exec --no-tty --no-key-check $(SETUP_NAME) "bash -c 'export DEBIAN_FRONTEND=noninteractive && $(APT_COMMAND) update && cat $(REQUIREMENTS_APT_ENV) | xargs -I % $(APT_COMMAND) install --no-install-recommends % && $(APT_COMMAND) clean && $(APT_COMMAND) autoremove && rm -rf /var/lib/apt/lists/*'"
-	neuro exec --no-tty --no-key-check $(SETUP_NAME) "bash -c '$(PIP_INSTALL_COMMAND) -r $(REQUIREMENTS_PIP_ENV)'"
-	neuro --network-timeout 300 job save $(SETUP_NAME) $(CUSTOM_ENV_NAME)
-	neuro kill $(SETUP_NAME)
+	$(NEURO) cp $(REQUIREMENTS_APT) $(REQUIREMENTS_APT_STORAGE)
```

Comment:
why not `$(NEURO_CP)` ?

Reply:
Because NEURO_CP refers to folder copying and therefore fails when working with files.